source/optional_features/custom_error_pages/index.html.md.erb

```diff
@@ -0,0 +1,24 @@
+---
+title: Custom error pages 
+weight: 135
+---
+
+# Custom error pages
```

Comment:
I don't think this is the right title. 'Custom' suggests you're customising something provided by the GOV.UK Pay service - which isn't the case. Also, in future we may offer the option of genuinely customising our unsuccessful payment pages (for example, allowing a service to have an optional paragraph) so this wouldn't leave us many options should we do that. We should have a look at the language that services which have requested this feature have used when talking about it.

Reply:
Well both you and Alex used the term "Use your own error pages" so perhaps we should try that?

Reply:
something like that, although i'd be careful with the word 'error' because that specifically describes one of the scenarios rather than the entire set. the scenarios are: 
1. payment declined 
2. payment cancelled 
3. payment expired 
4. error (eg technical error)
One service who requested it have called it 'payment failure screens', which (IMO) has a better ring to it than 'unsuccessful payment'.